src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectFile.cs

```diff
@@ -54,12 +54,12 @@ public string GetPropertyValue(string name)
 
         protected class BuildResult
         {
-            public readonly MSB.Execution.BuildResult Result;
+            public readonly bool Built;
```

Comment:
No, there aren't any usages of the `Result` field. In case the project is not actually built, there is nowhere to get this value from, so since it's unused I figured it's better replaced with this flag only.